CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,81 @@
+# Code of Conduct
+
+In order to deliver great features and support to the Renovate Open Source project, we ask for high politeness and careful attention from those who want to participate.
+
+## Our priorities
+
+Our main goal is to keep this project sustainable.
+This involves supporting our maintainers and contributors, most of whom volunteer their time to help others.
+The biggest threat to Open Source projects is when maintainers get too stressed, and this stress is often caused by rudeness from a few users.
```

Comment:
```suggestion
We want to keep this project sustainable.
This means we support our maintainers and contributors, who spend their free time to help others.

Maintainers getting stressed is a big threat to Open Source projects, like ours.
Stressed maintainers quit, or reduce their time spent on the project.
Often a few users behave badly, where most users are nice.

We want to avoid maintainers getting stressed out by bad behavior from contributors.
That's why we have these rules.
```